test/e2e/framework.go

```diff
@@ -2634,3 +2634,22 @@ func isConnectionLostError(err error) bool {
 func retryOnConnectionLostError(backoff wait.Backoff, fn func() error) error {
 	return retry.OnError(backoff, isConnectionLostError, fn)
 }
+
+func (data *TestData) checkAntreaAgentInfo(interval time.Duration, timeout time.Duration, name string) error {
+	err := wait.Poll(interval, timeout, func() (bool, error) {
+		aai, err := data.crdClient.CrdV1beta1().AntreaAgentInfos().Get(context.TODO(), name, metav1.GetOptions{})
+		if err != nil {
+			if errors.IsNotFound(err) {
+				return false, nil
+			} else {
```

Comment:
You can remove this `else` to reduce unnecessary indent since you will return in both cases.